opm/parser/eclipse/EclipseState/TracerConfig.hpp

```diff
@@ -0,0 +1,86 @@
+/*
+  Copyright (C) 2020 Equinor
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef OPM_TRACER_CONFIG_HPP
+#define OPM_TRACER_CONFIG_HPP
+
+#include <opm/parser/eclipse/EclipseState/Runspec.hpp>
+#include <opm/parser/eclipse/EclipseState/Tables/TracerVdTable.hpp>
+
+namespace Opm {
+
+class Deck;
+
+class TracerConfig {
+public:
+    struct TracerEntry {
+        std::string name;
+        Phase phase = Phase::OIL;
```

Comment:
I guess it has to be initialized to something - but I am quite certain `Phase::OIL` is *never* right.